content/chainguard/chainguard-images/working-with-images/unique-tags/index.md

```diff
@@ -43,17 +43,17 @@ Additionally, the Unique Tags feature is integrated with Chainguard's [Tag Histo
 
 After signing into the Chainguard Console, click on **Organization images** in the left-hand navigation. This will take you to your organization's Images directory where you'll be presented with a list of all the Chainguard Production Images your organization has access to.
 
-To the right of the search box and **Category** drop-down menu is a toggle button labeled **Include unique tags**. 
+To the right of the search box and **Category** drop-down menu there's a filter button labeled **Visible tags**. Click on that button, and you'll see a drop-down menu with two options: **Epoch tags** and **Unique tags**. Toggle **Unique tags** to see the Unique Tags available for your organization's Images.
 
-![Screenshot of the Organization images page, showing four Images in a table. The "Include unique tags" button is highlighted with a yellow box.](unique-tags-1.png)
+![Screenshot of the Organization images page, showing Images in a table. The "Visible Tags" select box is highlighted with a yellow box, showing the option "Unique tags" checked.](unique-tags-01.png)
 
 By toggling this button on, each individual Image's details page will show the Unique Tags available for it.
 
-To illustrate, toggle this button on and then click on any Production Image listed in your organization's directory. This example shows a partial list of available versions of the `busybox` Image from the **Versions** tab on its details page.
+To illustrate, toggle this button on and then click on any Production Image listed in your organization's directory. The "Version" column will now show the Unique Tags available for that Image. These tags include a timestamp in the format `YYYYMMDDHHMM`, and may include a prefix to help identify and parse the tag name programmatically.
 
-![Screenshot of a portion of the busybox Image's Versions tab. This screenshot shows the ten most recently built versions, some of which include unique tags such as "1-202312280512"](unique-tags-2.png)
+![This screenshot shows the ten most recently built versions of an image that has unique tags enabled.](unique-tags-02.png)
 
-Here there are a number of Image versions with tags similar to `:1-202312280512`. This means that this particular version of the Image was last updated on December 28, 2023 at 5:12 AM. You can use this version's **Pull URL** (`cgr.dev/$ORGANIZATION/busybox:1-202312280512`) to download this Image, and you can be confident that this Pull URL will always refer to the same Image.
+Here there are a number of Image versions with tags similar to `:openjdk-17-202412120223`. This means that this particular version of the Image was last updated on December 12, 2024, at 2:23 AM. You can use this version's **Pull URL** (`cgr.dev/$ORGANIZATION/jdk-fips:openjdk-17-202412120223`) to download this Image, and you can be confident that this Pull URL will always refer to the same Image.
 
 
 ## Accessing Unique Tags
```

Comment:
I wonder if it would make more sense to move this up higher. Like, we say it's an opt-in feature but then the walk through makes it seems like to opt in I just need to go to the console and check the right box, almost like we forget the initial step of contacting chainguard and getting access

Reply:
Yeah I was thinking about that, will update!

Reply:
Moved this to the intro!